crates/speedy-transform/src/web_transform/babel_import.rs

```diff
@@ -54,20 +58,41 @@ pub fn transform_style(
         for specifier in &var.specifiers {
           match specifier {
             ImportSpecifier::Named(ref s) => {
-              let ident: String = s.local.sym.to_string();
+              let imported = match &s.imported {
+                Some(imported) => match &imported {
+                  ModuleExportName::Ident(ident) => Option::Some(ident.sym.to_string()),
+                  ModuleExportName::Str(ident) => Option::Some(ident.value.to_string()),
+                },
+                None => Option::None,
+              };
+              // 当 imported 不为 none 时, local.sym 是引入组件的 as 别名
+              let as_name: Option<String> = if imported.is_some() {
+                Option::Some(s.local.sym.to_string())
+              } else {
+                Option::None
+              };
+              // 当 imported 不为 none 时, 实际引入的组件命名为 imported, 否则为 s.local.sym
+              let ident: String = imported.unwrap_or(s.local.sym.to_string());
+
               if match_ident(&s.local) {
                 // 替换对应的 css
                 if let Some(ref css) = child_config.replace_css {
                   let ignore_component = &css.ignore_style_component;
                   let need_lower = css.lower.unwrap_or(false);
-                  let css_ident = if need_lower {
-                    ident.to_lowercase()
-                  } else {
-                    ident.clone()
+                  let camel2dash = css.camel2_dash_component_name.unwrap_or(true);
+                  let mut css_ident = ident.clone();
+                  if camel2dash {
+                    css_ident = css_ident.to_kebab_case();
+                  }
+                  if need_lower {
+                    css_ident = css_ident.to_lowercase()
                   };
                   let mut need_replace = true;
                   if let Some(block_list) = ignore_component {
-                    need_replace = !block_list.iter().map(|c| c.as_str()).any(|x| x == ident);
+                    need_replace = !block_list
```

Comment:
This could be simplified as `!block_list.iter().any(|c| c == &css_ident)`

Reply:
fix